modules/logging-loki-logcli.adoc

```diff
@@ -0,0 +1,32 @@
+// Module is included in the following assemblies:
+// 
+//
+:_content-type: REFERENCE
+[id="logging-loki-logcli-about_{context}"]
+
+= Querying Loki
+
+You can use Loki's command-line interface `logcli` to query logs.
```

Comment:
```suggestion
You can use the Loki `logcli` CLI to query logs.
```
CLI doesn't need to be spelled out, CLI is used throughout OCP docs. See https://docs.openshift.com/container-platform/4.13/cli_reference/index.html

Can we add a bit more context here / use cases? Query logs for what? What does querying logs provide the user with?